File: PhoneWordFSharp/PhoneWordFSharp.Droid/MainActivity.fs
namespace PhoneWordFSharp.Droid

open System
open Android.App
open Android.Content
open Android.OS
open Android.Runtime
open Android.Views
open Android.Widget
open Xamarin.Forms
open MainPage
open OpenUrl
open Xamarin.Forms.Platform.Android

[<assembly: Dependency(typeof<OpenUrlService>)>] do ()

[<Activity (Label = "PhoneWordFSharp.Droid", MainLauncher = true)>]
type MainActivity () =
    inherit FormsApplicationActivity ()

    override this.OnCreate (bundle) =
        base.OnCreate (bundle)

        Forms.Init(this, bundle)

        this.LoadApplication(App())

File: DisclosureAccessoryDemo/DisclosureAccessoryDemo/DisclosureAccessoryDemo.Droid/MainActivity.cs
using Android.App;
using Android.Content.PM;
using Android.OS;

namespace DisclosureAccessoryDemo.Droid
{
    [Activity(
        Label = "DisclosureAccessoryDemo",
        Icon = "@drawable/icon",
        MainLauncher = true,
        ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation,
        Theme = "@style/CustomTheme")]
    public class MainActivity : global::Xamarin.Forms.Platform.Android.FormsApplicationActivity
    {
        protected override void OnCreate(Bundle bundle)
        {
            base.OnCreate(bundle);

            global::Xamarin.Forms.Forms.Init(this, bundle);
            LoadApplication(new App());
        }
    }
}

